doc/application_workflow/harbor_registry/harbor_with_static_ip.md

```diff
@@ -0,0 +1,209 @@
+# Installing Harbor with static IP for use with vSphere Integrated Container Engine
+
+For testing and development purposes, it is often useful to simply deploy Harbor with a static IP.  This workflow will demonstrate how to do that.  The primary difference in this workflow versus the one for FQDN will be in the vApp parameters for Harbor and the certificate generation section.  When installing Harbor with static IP, we will install Harbor with networking and the certificate in one step instead of the two steps that we listed for FQDN.  Note, our install for FQDN required two steps to accomodate our setup.  other VC setups may only require 1 step.
+<br><br>
+
+## Workflow
+
+We will use VIC Engine 0.8.0 and Harbor 0.4.5 for this example.  We will also use Windows during the install of Harbor as we will use the vSphere thick client for Windows.  We will use Ubuntu as OS on our user machine for every other step.
+
+By the end of this, we should have demonstrated,
+
+1. Self-generating a CA certificate, a server certificate, and a server private key for a Harbor instance with static IP
+2. Installing Habor on a vCenter installation
+3. Updating a standard docker daemon with the CA certificate
+4. Install a VCH with the new CA certificate
+<br><br>
+
+## Generate a CA certificate, a server certificate, and a server private key with a static IP
+
+There are many examples and instructions on generating certificates using openssl.  Here are documentation from the Harbor's docs on [setting up https](https://github.com/vmware/harbor/blob/master/docs/configure_https.md).  Hare are instructions from Docker's docs for [docker engine security](https://docs.docker.com/engine/security/https/).  Both use openssl and are similar.  We will use the one from the Harbor docs.  At the end, there will be 3 files that we will need: ca.crt, server.crt, server.key.  We will install the server cert and key in Harbor.  We will deploy a VCH with the CA cert.
```

Comment:
Maybe replace `here are documentation from the Harbor's docs` with 'Here is Harbor's documentation`, or something similar.